draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
 
 # Summary
 
-[summary]: #summary
-
-Currently there the Rust Programming Language organisation has a set teams called "Working Groups", however the definition of what these are has become ill defined since their initial creation, especially as more and more people have used the same moniker for different purposes. This has caused quite a bit of confusion between team members and the community at large. This RFC seeks to clarify and codify the different sets of groups previously under the "Working Group" umbrella term.
+Currently the Rust Programming Language organisation has a set of teams
```

Comment:
I tend to favor "bullet point" summaries that try to capture the important changes being proposed (and in particular that do not contain much motivation). I took a stab in [this hackmd](https://hackmd.io/xfW6jj7DQXeeVeNoCW2g3g) at defining some of the key points from this RFC, inlined below. Maybe something like this?

* Formalize **project groups** as groups dedicated to completing specific projects within the context of a Rust team
    * Project groups are created via an RFC and have a "parent team" (or multiple teams)
    * The groups then drive the project to completion, e.g. by authoring follow-up RFCs and doing design work
    * Each project group typically has:
        * charter
        * defined shepherds and team liasons
        * an associated repository
        * regular meetings
        * dedicated streams on Discord/Zulip/etc
* Define **working groups** to refer to the "domain working groups" that are created to explore particular domains, such as embedded, CLI, etc
    * They have a charter and defined leads but operate more independently from the Rust teams
* Define **community group** as the term for groups not formally affiliated with the Rust project